### DISCUSSION TITLE
Java Template doesn't automatically create an Endpoint

### DISCUSSION THREAD

**Discussion by 1800joe** [noop-inc/documentation#142] [Bugs]
- Use the latest version of the Java Template.
- Automatically start Env
- Everything comes online, except the Endpoint

Workshop v0.10.3

**Comment by derekwolpert** [comment]
@1800joe I cannot replicate this issue on my end. Any further context you can offer here?

<img width="1392" alt="Screenshot 2024-03-22 at 2 20 46 PM" src="https://github.com/noop-inc/documentation/assets/48101033/72dad7d9-2879-404c-b155-445298514316">

**Comment by 1800joe** [comment]
Sure, what would be helpful?

FWIW it doesn't seem limited to Java Template. I'm seeing this on astro and django templates as well.

![Screenshot 2024-03-22 at 3 11 20 PM](https://github.com/noop-inc/documentation/assets/3111670/732acaf2-f859-4e52-9ad3-9e00fa91d1cc)
